Add `FDC 2025` tournament page (#12944)
ppy/osu-wiki: wiki/Tournaments/FDC/2025/en.md

| Mandelbrot set | ::{ flag=FI }:: **[Jokupoika24](https://osu.ppy.sh/users/17782643)**, ::{ flag=FI }:: [System Sun](https://osu.ppy.sh/users/3163106) |
| Lil bros | ::{ flag=FI }:: **[ImTK](https://osu.ppy.sh/users/11506206)**, ::{ flag=FI }:: [OG_PannuLappu](https://osu.ppy.sh/users/7295343) |

## Podium

This competition has come to an end and resulted in the following podium:

| Placing | Team |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | **oddloop** (::{ flag=FI }:: [Kalanluu](https://osu.ppy.sh/users/2035254), ::{ flag=PL }:: [MALISZEWSKI](https://osu.ppy.sh/users/12408961) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | **bunny party** (::{ flag=FI }:: [Eevert](https://osu.ppy.sh/users/12080544), ::{ flag=KR }:: [Amamya Kokoro](https://osu.ppy.sh/users/2511839) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | **terence** (::{ flag=FI }:: [house](https://osu.ppy.sh/users/15242810), ::{ flag=NO }:: [NINERIK](https://osu.ppy.sh/users/10549880) |

## Mappools

### Grand Finals

> **[Download the mappack here! (206 MB)](https://drive.google.com/file/d/1b1bOrRfA6KdWRS1n5-K14Wkk8kQr2Z7n/view?usp=drive_link)**

Reviewer: @peppy just to confirm, since you [mentioned](https://github.com/ppy/osu-wiki/pull/13098#discussion_r2030443422) that we shouldn't use external download links, was that concerning world cups only or the entire wiki?

tournaments have all included mappack downloads since forever, though they're more so useful to players during ongoing tournaments

Reviewer: can be handled as a follow-up if needed

- Double Time
  1. [Mazare & Philip Strand - Battlecry (Mako) [Fervor of Battle]](https://osu.ppy.sh/beatmapsets/1167000#osu/2434310)
  2. [Christina Vee - Caramelldansen (ft. RichaadEB) (xidorn) [forsta era tangentborden]](https://osu.ppy.sh/beatmapsets/2101527#osu/4408741)
  3. [BlackYooh vs. siromaru - BLACK or WHITE? (BluOxy) [INFINITE]](https://osu.ppy.sh/beatmapsets/248876#osu/637477)
  4. [CLIFF EDGE feat. Nakamura Maiko - The Distance (handsome) [Master]](https://osu.ppy.sh/beatmapsets/580375#osu/1228840)
- Free Mod
  1. [Reol - Gigantic O.T.N (sparhten) [Climax]](https://osu.ppy.sh/beatmapsets/1276362#osu/2651849)
  2. [Rings Of Saturn - Lalassu Xul (Val) [KurtZenN's Extreme]](https://osu.ppy.sh/beatmapsets/1560427#osu/3677314)
  3. [Down - DeviousPanda (Down) [Extra]](https://osu.ppy.sh/beatmapsets/1789325#osu/3666374)
  4. [wowaka - World's End Dancehall (Deppyforce) [Yokes' Expert]](https://osu.ppy.sh/beatmapsets/1737165#osu/3727749)
- Tiebreaker
  1. **[Ludicin - Echoes of Memoria (hehe) [Hypermnesia]](https://osu.ppy.sh/beatmapsets/2214238#osu/4691422)**

>   ### Finals

Reviewer: ```suggestion
### Finals
```

---
tags:
  - FDC2025
  - FDC
---

# Finnish Duo Cup 2025

![Finnish Duo Cup 2025 logo](img/banner.png)

> The **Finnish Duo Cup 2025** (**FDC2025**) was a 2v2 double-elimination osu!standard tournament hosted by ::{ flag=FI }:: [Aetherian](https://osu.ppy.sh/users/6263040). The tournament required at least one of the players in each team to be from ::{ flag=FI }:: Finland or ::{ flag=AX }:: Åland. It was the third instalment of the Finnish Duo Cup.

Reviewer: official name of the game mode is just "osu!"

```suggestion
The **Finnish Duo Cup 2025** (**FDC2025**) was a 2v2 double-elimination osu! tournament hosted by ::{ flag=FI }:: [Aetherian](https://osu.ppy.sh/users/6263040). The tournament required at least one of the players in each team to be from ::{ flag=FI }:: Finland or ::{ flag=AX }:: Åland. It was the third instalment of the Finnish Duo Cup.
```

  3. [KARUT - Glitch Street (Tycani) [Fallen Eden (Circle Circuit Championships Ver.)]](https://osu.ppy.sh/beatmapsets/1854718#osu/3811238)
- Double Time
  1. [Mazare & Philip Strand - Battlecry (Mako) [Fervor of Battle]](https://osu.ppy.sh/beatmapsets/1167000#osu/2434310)
  2. [Christina Vee - Caramelldansen (ft. RichaadEB) (xidorn) [forsta era tangentborden]](https://osu.ppy.sh/beatmapsets/2101527#osu/4408741)
  3. [BlackYooh vs. siromaru - BLACK or WHITE? (BluOxy) [INFINITE]](https://osu.ppy.sh/beatmapsets/248876#osu/637477)
  4. [CLIFF EDGE feat. Nakamura Maiko - The Distance (handsome) [Master]](https://osu.ppy.sh/beatmapsets/580375#osu/1228840)
- Free Mod
  1. [Reol - Gigantic O.T.N (sparhten) [Climax]](https://osu.ppy.sh/beatmapsets/1276362#osu/2651849)
  2. [Rings Of Saturn - Lalassu Xul (Val) [KurtZenN's Extreme]](https://osu.ppy.sh/beatmapsets/1560427#osu/3677314)
  3. [Down - DeviousPanda (Down) [Extra]](https://osu.ppy.sh/beatmapsets/1789325#osu/3666374)
  4. [wowaka - World's End Dancehall (Deppyforce) [Yokes' Expert]](https://osu.ppy.sh/beatmapsets/1737165#osu/3727749)
- Tiebreaker

> 

Reviewer: ```suggestion
```

  3. [Towa Tei with Sheena Ringo - APPLE (fanzhen0019) [PaRaDogi's Expert]](https://osu.ppy.sh/beatmapsets/1851719#osu/3941000)
- Double Time
  1. [Jager - I Won't Fall Apart (Vell) [Insane]](https://osu.ppy.sh/beatmapsets/497154#osu/1058295)
  2. [senya - Shunkan Everlasting (Satellite) [Lunatic]](https://osu.ppy.sh/beatmapsets/1981457#osu/4114463)
  3. [cosMo@BousouP feat. Hatsune Miku - Zero kara Eien e no Choudou (P A N) [Insane]](https://osu.ppy.sh/beatmapsets/1718717#osu/3514231)
  4. [android52 - Platinum Groove (Sinnoh) [Extra Extra Extra Extra]](https://osu.ppy.sh/beatmapsets/848144#osu/1863918)
- Free Mod
  1. [Yooh - Destroy -agitato- (Realazy) [Extreme]](https://osu.ppy.sh/beatmapsets/839273#osu/1756964)
  2. [FLESHGOD APOCALYPSE - The Fool / Thru Our Scars (Fenza) [The Fool]](https://osu.ppy.sh/beatmapsets/418316#osu/1177658)
  3. [Rin - Kishinjou set 03 ~ Humans and Youkai Traversing the Canal (yaspo) [hasplasho's Human x Youkai CoLunatic]](https://osu.ppy.sh/beatmapsets/1799358#osu/3688644)
  4. [dandeless - micro.wav (yaspo) [USA Tryouts]](https://osu.ppy.sh/beatmapsets/1499766#osu/3198204)
- Tiebreaker

> 

Reviewer: ```suggestion
```

  1. **[HyuN - Duplicity Shade (feat. Sennzai) (Down) [Salvation]](https://osu.ppy.sh/beatmapsets/1708718#osu/3491545)**

>   ### Semifinals

Reviewer: ```suggestion
### Semifinals
```

  3. [BEMANI Sound Team "Nekomata Master" - Painful Fate (Down) [Cub's Extra]](https://osu.ppy.sh/beatmapsets/1193072#osu/2486379)
- Double Time
  1. [Toyosaki Aki - Diamond (Fycho) [Insane]](https://osu.ppy.sh/beatmapsets/111760#osu/290581)
  2. [cygnus - Violet SunLight (Sanch-KK) [Insane]](https://osu.ppy.sh/beatmapsets/2152760#osu/4545563)
  3. [cosMo@BousouP - 25-ji 76-pun Hatsu \<\<Ryouiki Gai\>\> Iki Saishuu Ressha (Sagon) [Insane]](https://osu.ppy.sh/beatmapsets/1807738#osu/3741963)
  4. [Faun - Tanz mit mir (kwk) [Cute Russian Song]](https://osu.ppy.sh/beatmapsets/802989#osu/1685542)
- Free Mod
  1. [Falcom Sound Team jdk - Don't be Defeated by a Friend! (Deca) [Lionheart]](https://osu.ppy.sh/beatmapsets/1557286#osu/3181298)
  2. [69 de 74 - DESPERATE DEATHPARADE (rollpan) [EXTRA]](https://osu.ppy.sh/beatmapsets/1939881#osu/4010877)
  3. [96neko - Buriki no Dance (CallieCube) [Tango Sugar Free]](https://osu.ppy.sh/beatmapsets/1332410#osu/4228080)
  4. [beatMARIO - Night of Knights (Anxient) [TRUE 2009 STAGE]](https://osu.ppy.sh/beatmapsets/530702#osu/1125657)
- Tiebreaker

> 

Reviewer: ```suggestion
```

  1. **[Asatsumei ft. L4hee - Tenshi no Kikyou (iljaaz) [Spectrum]](https://osu.ppy.sh/beatmapsets/2278590#osu/4856643)**

>   ### Quarterfinals

Reviewer: ```suggestion
### Quarterfinals
```

  3. [MomoKuro-tei Ichimon - Nippon Egao Hyakkei (TV Size) (CXu) [Egao]](https://osu.ppy.sh/beatmapsets/74301#osu/211154)
- Double Time
  1. [Motley Croe - Kickstart My Heart (Castagne) [sasas]](https://osu.ppy.sh/beatmapsets/1332159#osu/2760226)
  2. [Katakiri Rekka - Tamayura (JAKAZiD's AKABEi HARD5 Remix) (niii_san) [Reiji Maigo's Insane]](https://osu.ppy.sh/beatmapsets/1323734#osu/2745951)
  3. [Sayuri - Tsuki to Hanataba (Mirash) [Insane]](https://osu.ppy.sh/beatmapsets/780885#osu/1641266)
  4. [Yuuyu - Akutagawa Ryuunosuke no Kappa ~ Candid Friend (Halfslashed) [Collab Insane]](https://osu.ppy.sh/beatmapsets/1703912#osu/4042385)
- Free Mod
  1. [TAROLIN feat. Katakiri Rekka - Cardinal Rave (Yusomi) [Expert]](https://osu.ppy.sh/beatmapsets/1116399#osu/2361457)
  2. [Nor - FEEEEVER TIME (Matsuyuki Ame) [Anxient's EXTRA]](https://osu.ppy.sh/beatmapsets/2076534#osu/4365526)
  3. [Scott Brown - Go Berzerk (Luscent) [Black Another]](https://osu.ppy.sh/beatmapsets/1596950#osu/3261566)
  4. [Helblinde - Grief & Malice (Okoratu) [Local's Expert]](https://osu.ppy.sh/beatmapsets/1846720#osu/3994108)
- Tiebreaker

> 

Reviewer: ```suggestion
```

  1. **[ToYou & DJ Raisei - ADHD (Down) [KCSC GRAND FINAL]](https://osu.ppy.sh/beatmapsets/1948527#osu/4033196)**

>   ### Round of 16

Reviewer: ```suggestion
### Round of 16
```

  2. [IOSYS - Osaisen <3 Choudai - Nonstop NEW RICH shrine mix - (Bloxi) [Nogard's SOLO]](https://osu.ppy.sh/beatmapsets/2051668#osu/4292547)
- Hard Rock
  1. [Mili - String Theocracy ([Arc]) [Theatrically Interesting]](https://osu.ppy.sh/beatmapsets/1192164#osu/2484169)
  2. [Doomsday - Length of Location (Doomsday) [Yokes' Expert]](https://osu.ppy.sh/beatmapsets/1767661#osu/3659336)
- Double Time
  1. [Misun - Eli Eli (Maduk Remix) (elicz1) [cz1 cz1]](https://osu.ppy.sh/beatmapsets/1654092#osu/3376247)
  2. [07th Expansion - Akai Tsukunise (soulfear) [Souken no Tou]](https://osu.ppy.sh/beatmapsets/14208#osu/52128)
  3. [Street - Maritime Police (Mir) [Lasse's Insane]](https://osu.ppy.sh/beatmapsets/681410#osu/1445844)
- Free Mod
  1. [Shoichiro Sakamoto - Eiyuu no Tsurugi (audinor) [Extra]](https://osu.ppy.sh/beatmapsets/1864097#osu/4468656)
  2. [xi - Heavenly Blast (2022 Remaster) (Bloxi) [ninore's Another]](https://osu.ppy.sh/beatmapsets/1987556#osu/4130247)
  3. [Aiko Kayo - Hitomi no Naka no Meikyuu (Reiji Maigo) [Insane+]](https://osu.ppy.sh/beatmapsets/1933416#osu/3994999)
- Tiebreaker

> 

Reviewer: ```suggestion
```

  1. **[yanaginagi - Outsider (apoq) [Eternity]](https://osu.ppy.sh/beatmapsets/1368755#osu/2844390)**

>   ### Round of 32

Reviewer: ```suggestion
### Round of 32
```

| **Meow & Myouow :3** | **0** | -1 | bouquet | *win by default* |
| São Paulo | 2 | **5** | **NO CLUE!** | [#1](https://osu.ppy.sh/community/matches/116873391) |

Saturday, 18 January 2025

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **might miss a match** | **5** | 0 | Schizo rizzzlers | [#1](https://osu.ppy.sh/community/matches/116856976) |
| **Karjalanpiirakka** | **5** | 0 | drillaz | [#1](https://osu.ppy.sh/community/matches/116857752) |
| **Suklaapallit** | **5** | 1 | Airline food | [#1](https://osu.ppy.sh/community/matches/116859311) |
| **ballers will ball** | **5** | 0 | Anttola honor | [#1](https://osu.ppy.sh/community/matches/116862570) |

Thursday, 16 January 2025

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **mä rakastan sua** | **1** | -1 | No title | [#1](https://osu.ppy.sh/community/matches/116834912) |

## Ruleset

### General

- This tournament is a 2v2 double-elimination osu! tournament.
- Each team consists of 2 players.
- At least 1 player from each team has to be from ::{ flag=FI }:: Finland or ::{ flag=AX }:: Åland.

> - Each team's total pp sum is capped at is 30,000pp, with maximum of 17,500pp per player.

Reviewer: ```suggestion
- Each team's total performance point sum is capped at is 30,000 pp, with maximum of 17,500 pp per player.
```

- Tournament uses custom `bws pp` formula for calculating accurate pp value for each player: `bws pp = pp ^ (1.001 ^ (badge_count ^2))`.

> - Each player can only contribute 17,500pp towards their team's pp sum meaning that if player has more than 17,500pp, only 17,500pp is counted towards the sum.

Reviewer: ```suggestion
- Each player can only contribute 17,500 pp towards their team's pp sum, meaning that if player has more than 17,500 pp, only 17,500 pp is counted towards the sum.
```

- All matches will use Score V2 and NF will be enforced on every map.
- All players have to treat staff members and all other players with a high level of respect. Any kind of mistreatment may result in being banned from the tournament.
- This is an open rank tournament.
- Staff members are not allowed to participate in the tournament.
- Every player is required to join the tournament Discord server.
- All times will be in UTC.

### Mappools

- There will be a new mappool every week.
- Mappools follow the following format:

| Stage | Maps |
| :-- | :-- |
| Qualifiers | 4 NM, 2 HD, 2 HR, 2 DT |
| Round of 32 | 4 NM, 2 HD, 2 HR, 3 DT, 3 FM, 1 TB |
| Round of 16 | 4 NM, 2 HD, 2 HR, 3 DT, 3 FM, 1 TB |
| Quarterfinals | 5 NM, 3 HD, 3 HR, 4 DT, 4 FM, 1 TB |
| Semifinals | 5 NM, 3 HD, 3 HR, 4 DT, 4 FM, 1 TB |
| Finals | 5 NM, 3 HD, 3 HR, 4 DT, 4 FM, 1 TB |
| Grand Finals | 5 NM, 3 HD, 3 HR, 4 DT, 4 FM, 1 TB |

### Qualifiers

- The Qualifier mappool consists of 4 NM, 2 HD, 2 HR, and 2 DT maps.
- The Qualifier mappool will be played in following order: NM -> DT.
- NF and Score V2 will be used on every map.
- The mappool will be played once only.

> - If a player disconnects during a map they can replay it after all other maps have been played and all other players have left the lobby.

Reviewer: ```suggestion
- If a player disconnects during a map, they can replay it after all other maps have been played and all other players have left the lobby.
```

|   **Melon Boys** | **6** |   2   | NATION                | [#1](https://osu.ppy.sh/community/matches/117103467) |
| Karjalanpiirakka |   1   | **6** | **NO CLUE!**          | [#1](https://osu.ppy.sh/community/matches/117104340) |
|    skibidi vessa |  -1   | **0** | **ballers will ball** | _win by default_                                     |

Wednesday, 5 February 2025

|      Team 1 |       |     | Team 2  | Match link                                           |
| ----------: | :---: | :-: | :------ | :--------------------------------------------------- |
| **oddloop** | **6** |  1  | terence | [#1](https://osu.ppy.sh/community/matches/117069912) |

### Quarterfinals

Monday, 3 February 2025

|           Team 1 |       |       | Team 2            | Match link                                           |
| ---------------: | :---: | :---: | :---------------- | :--------------------------------------------------- |
|      **oddloop** | **6** |   1   | mä rakastan sua   | [#1](https://osu.ppy.sh/community/matches/117047568) |
|      Gorillamode |   1   | **6** | **carrot fields** | [#1](https://osu.ppy.sh/community/matches/117048430) |

> | **Suklaapallit** | **6** |   3   | Karjalanpiirakka  | [#1](https://osu.ppy.sh/community/matches/117048947) |

Reviewer 2: ```suggestion
| **Suklaapallit** | **6** | 3 | Karjalanpiirakka | [#1](https://osu.ppy.sh/community/matches/117048947) |
```

Sunday, 2 February 2025

> |                Team 1 |       |       | Team 2            | Match link                                           |

Reviewer 2: ```suggestion
| Team 1 |  |  | Team 2 | Match link |
```

> | --------------------: | :---: | :---: | :---------------- | :--------------------------------------------------- |

Reviewer 2: ```suggestion
| --: | :-: | :-: | :-- | :-- |
```

> |       **bunny party** | **6** |   3   | Melon boys        | [#1](https://osu.ppy.sh/community/matches/117036745) |

Reviewer 2: ```suggestion
| **bunny party** | **6** | 3 | Melon boys | [#1](https://osu.ppy.sh/community/matches/117036745) |
```

> | **ballers will ball** | **6** |   3   | Galaxy Destroyers | [#1](https://osu.ppy.sh/community/matches/117037811) |

Reviewer 2: ```suggestion
| **ballers will ball** | **6** | 3 | Galaxy Destroyers | [#1](https://osu.ppy.sh/community/matches/117037811) |
```

> |    might miss a match |   5   | **6** | **NO CLUE!**      | [#1](https://osu.ppy.sh/community/matches/117038110) |

Reviewer 2: ```suggestion
| might miss a match | 5 | **6** | **NO CLUE!** | [#1](https://osu.ppy.sh/community/matches/117038110) |
```

|           **terence** | **6** |   1   | skibidi vessa     | [#1](https://osu.ppy.sh/community/matches/117038819) |
|          Airline food |   0   | **6** | **NATION**        | [#1](https://osu.ppy.sh/community/matches/117041154) |

Saturday, 1 February 2025

> |                 Team 1 |       |       | Team 2           | Match link                                           |

Reviewer 2: ```suggestion
| Team 1 |  |  | Team 2 | Match link |
```

| ---------------------: | :---: | :---: | :--------------- | :--------------------------------------------------- |
|        **Gorillamode** | **6** |   1   | 4444             | [#1](https://osu.ppy.sh/community/matches/117021057) |
|  **Galaxy Destroyers** | **0** |  -1   | Schizo rizzzlers | _win by default_                                     |

> | **might miss a match** | **6** |   2   | happy new year   | [#1](https://osu.ppy.sh/community/matches/117021297) |

Reviewer 2: ```suggestion
| **might miss a match** | **6** | 2 | happy new year | [#1](https://osu.ppy.sh/community/matches/117021297) |
```

> |       Meow & Myouow :3 |  -1   | **0** | **Airline food** | _win by default_                                     |

Reviewer 2: ```suggestion
| Meow & Myouow :3 | -1 | **0** | **Airline food** | _win by default_ |
```

> |           **NO CLUE!** | **6** |   0   | Anttola honor    | [#1](https://osu.ppy.sh/community/matches/117025857) |

Reviewer 2: ```suggestion
| **NO CLUE!** | **6** | 0 | Anttola honor | [#1](https://osu.ppy.sh/community/matches/117025857) |
```

> |  **ballers will ball** | **6** |   4   | São Paulo        | [#1](https://osu.ppy.sh/community/matches/117027027) |

Reviewer 2: ```suggestion
| **ballers will ball** | **6** | 4 | São Paulo | [#1](https://osu.ppy.sh/community/matches/117027027) |
```

Friday, 31 January 2025

> |     Team 1 |       |     | Team 2         | Match link                                           |

Reviewer 2: ```suggestion
| Team 1 |  |  | Team 2 | Match link |
```

> | ---------: | :---: | :-: | :------------- | :--------------------------------------------------- |

Reviewer 2: ```suggestion
| --: | :-: | :-: | :-- | :-- |
```

> | **NATION** | **6** |  0  | Mandelbrot set | [#1](https://osu.ppy.sh/community/matches/117009120) |

Reviewer 2: ```suggestion
| **NATION** | **6** | 0 | Mandelbrot set | [#1](https://osu.ppy.sh/community/matches/117009120) |
```

Tuesday, 28 January 2025

> |            Team 1 |       |     | Team 2  | Match link                                           |

Reviewer 2: ```suggestion
| Team 1 |  |  | Team 2 | Match link |
```

> | ----------------: | :---: | :-: | :------ | :--------------------------------------------------- |

Reviewer 2: ```suggestion
| --: | :-: | :-: | :-- | :-- |
```

| **carrot fields** | **6** |  0  | bouquet | [#1](https://osu.ppy.sh/community/matches/116979149) |

### Round of 16

Monday, 27 January 2025

> |              Team 1 |       |     | Team 2             | Match link                                           |

Reviewer 2: ```suggestion
| Team 1 |  |  | Team 2 | Match link |
```

> | ------------------: | :---: | :-: | :----------------- | :--------------------------------------------------- |

Reviewer 2: ```suggestion
| --: | :-: | :-: | :-- | :-- |
```

| **mä rakastan sua** | **0** | -1  | might miss a match | _win by default_                                     |

> |   **skibidi vessa** | **5** |  2  | Meow & Myouow :3   | [#1](https://osu.ppy.sh/community/matches/116969160) |

Reviewer 2: ```suggestion
| **skibidi vessa** | **5** |   | Meow & Myouow :3 | [#1](https://osu.ppy.sh/community/matches/116969160) |
```

Sunday, 26 January 2025

> |         Team 1 |       |       | Team 2               | Match link                                           |

Reviewer 2: ```suggestion
| Team 1 |  |  | Team 2 | Match link |
```

> | -------------: | :---: | :---: | :------------------- | :--------------------------------------------------- |

Reviewer 2: ```suggestion
| --: | :-: | :-: | :-- | :-- |
```

> |    Gorillamode |   2   | **5** | **Karjalanpiirakka** | [#1](https://osu.ppy.sh/community/matches/116954236) |

Reviewer 2: ```suggestion
| Gorillamode | 2 | **5** | **Karjalanpiirakka** | [#1](https://osu.ppy.sh/community/matches/116954236) |
```

> |  carrot fields |   4   | **5** | **Suklaapallit**     | [#1](https://osu.ppy.sh/community/matches/116956998) |

Reviewer 2: ```suggestion
| carrot fields | 4 | **5** | **Suklaapallit** | [#1](https://osu.ppy.sh/community/matches/116956998) |
```

> | **Melon boys** | **5** |   2   | ballers will ball    | [#1](https://osu.ppy.sh/community/matches/116958610) |

Reviewer 2: ```suggestion
| **Melon boys** | **5** | 2 | ballers will ball | [#1](https://osu.ppy.sh/community/matches/116958610) |
```

Saturday, 25 January 2025

> |             Team 1 |       |       | Team 2               | Match link                                           |

Reviewer 2: ```suggestion
| Team 1 |  |  | Team 2 | Match link |
```

| -----------------: | :---: | :---: | :------------------- | :--------------------------------------------------- |

> |        **oddloop** | **5** |   1   | NO CLUE!             | [#1](https://osu.ppy.sh/community/matches/116942008) |

Reviewer 2: ```suggestion
| **oddloop** | **5** | 1 | NO CLUE! | [#1](https://osu.ppy.sh/community/matches/116942008) |
```

> |    **bunny party** | **5** |   1   | Galaxy Destroyers    | [#1](https://osu.ppy.sh/community/matches/116942513) |

Reviewer 2: ```suggestion
| **bunny party** | **5** | 1 | Galaxy Destroyers | [#1](https://osu.ppy.sh/community/matches/116942513) |
```

> | **Mandelbrot set** | **5** |   0   | drillaz              | [#1](https://osu.ppy.sh/community/matches/116942826) |

Reviewer 2: ```suggestion
| **Mandelbrot set** | **5** | 0 | drillaz | [#1](https://osu.ppy.sh/community/matches/116942826) |
```

|           No title |   2   | **5** | **Schizo rizzzlers** | [#1](https://osu.ppy.sh/community/matches/116943225) |

> |           Lil bros |   0   | **5** | **Anttola honor**    | [#1](https://osu.ppy.sh/community/matches/116943666) |

Reviewer 2: ```suggestion
| Lil bros | 0 | **5** | **Anttola honor** | [#1](https://osu.ppy.sh/community/matches/116943666) |
```

> |        **terence** | **5** |   1   | NATION               | [#1](https://osu.ppy.sh/community/matches/116943668) |

Reviewer 2: ```suggestion
| **terence** | **5** | 1 | NATION | [#1](https://osu.ppy.sh/community/matches/116943668) |
```

### Round of 32

Sunday, 19 January 2025

> |                Team 1 |       |       | Team 2         | Match link                                           |

Reviewer 2: ```suggestion
| Team 1 |  |  | Team 2 | Match link |
```

| --------------------: | :---: | :---: | :------------- | :--------------------------------------------------- |
| **Galaxy Destroyers** | **5** |   1   | happy new year | [#1](https://osu.ppy.sh/community/matches/116869405) |

> |       **Gorillamode** | **5** |   3   | Mandelbrot set | [#1](https://osu.ppy.sh/community/matches/116870277) |

Reviewer 2: ```suggestion
| **Gorillamode** | **5** | 3 | Mandelbrot set | [#1](https://osu.ppy.sh/community/matches/116870277) |
```

> |        **Melon boys** | **5** |   0   | Lil bros       | [#1](https://osu.ppy.sh/community/matches/116871232) |

Reviewer 2: ```suggestion
| **Melon boys** | **5** | 0 | Lil bros | [#1](https://osu.ppy.sh/community/matches/116871232) |
```

> |            **NATION** | **5** |   2   | 4444           | [#1](https://osu.ppy.sh/community/matches/116871992) |

Reviewer 2: ```suggestion
| **NATION** | **5** | 2 | 4444 | [#1](https://osu.ppy.sh/community/matches/116871992) |
```

> |  **Meow & Myouow :3** | **0** |  -1   | bouquet        | _win by default_                                     |

Reviewer 2: ```suggestion
| **Meow & Myouow :3** | **0** | -1 | bouquet | _win by default_ |
```

|             São Paulo |   2   | **5** | **NO CLUE!**   | [#1](https://osu.ppy.sh/community/matches/116873391) |

Saturday, 18 January 2025

> |                 Team 1 |       |     | Team 2           | Match link                                           |

Reviewer 2: ```suggestion
| Team 1 |  |  | Team 2 | Match link |
```

| ---------------------: | :---: | :-: | :--------------- | :--------------------------------------------------- |
| **might miss a match** | **5** |  0  | Schizo rizzzlers | [#1](https://osu.ppy.sh/community/matches/116856976) |

> |   **Karjalanpiirakka** | **5** |  0  | drillaz          | [#1](https://osu.ppy.sh/community/matches/116857752) |

Reviewer 2: ```suggestion
| **Karjalanpiirakka** | **5** | 0 | drillaz | [#1](https://osu.ppy.sh/community/matches/116857752) |
```

|       **Suklaapallit** | **5** |  1  | Airline food     | [#1](https://osu.ppy.sh/community/matches/116859311) |

> |  **ballers will ball** | **5** |  0  | Anttola honor    | [#1](https://osu.ppy.sh/community/matches/116862570) |

Reviewer 2: ```suggestion
| **ballers will ball** | **5** | 0 | Anttola honor | [#1](https://osu.ppy.sh/community/matches/116862570) |
```

Thursday, 16 January 2025

> |              Team 1 |       |     | Team 2   | Match link                                           |

Reviewer 2: ```suggestion
| Team 1 |  |  | Team 2 | Match link |
```

> | ------------------: | :---: | :-: | :------- | :--------------------------------------------------- |

Reviewer 2: ```suggestion
| --: | :-: | :-: | :-- | :-- |
```

> | **mä rakastan sua** | **1** | -1  | No title | [#1](https://osu.ppy.sh/community/matches/116834912) |

Reviewer 2: ```suggestion
| **mä rakastan sua** | **1** | -1 | No title | [#1](https://osu.ppy.sh/community/matches/116834912) |
```

## Ruleset

### General

- This tournament is a 2v2 double-elimination osu! tournament.
- Each team consists of 2 players.
- At least 1 player from each team has to be from ::{ flag=FI }:: Finland or ::{ flag=AX }:: Åland.
- Each team's total pp sum is capped at is 30,000pp, with maximum of 17,500pp per player.
- Tournament uses custom `bws pp` formula for calculating accurate pp value for each player: `bws pp = pp ^ (1.001 ^ (badge_count ^2))`.
- Each player can only contribute 17,500pp towards their team's pp sum meaning that if player has more than 17,500pp, only 17,500pp is counted towards the sum.
- All matches will use Score V2 and NF will be enforced on every map.
- All players have to treat staff members and all other players with a high level of respect. Any kind of mistreatment may result in being banned from the tournament.
- This is an open rank tournament.
- Staff members are not allowed to participate in the tournament.
- Every player is required to join the tournament Discord server.
- All times will be in UTC.

### Mappools

- There will be a new mappool every week.
- Mappools follow the following format:

| Stage         | Maps                               |

> | :------------ | :--------------------------------- |

Reviewer 2: ```suggestion
| :-- | :-- |
```

| Qualifiers    | 4 NM, 2 HD, 2 HR, 2 DT             |

> | Round of 32   | 4 NM, 2 HD, 2 HR, 3 DT, 3 FM, 1 TB |

Reviewer 2: ```suggestion
| Round of 32 | 4 NM, 2 HD, 2 HR, 3 DT, 3 FM, 1 TB |
```

> | Round of 16   | 4 NM, 2 HD, 2 HR, 3 DT, 3 FM, 1 TB |

Reviewer 2: ```suggestion
| Round of 16 | 4 NM, 2 HD, 2 HR, 3 DT, 3 FM, 1 TB | 
```

> | Quarterfinals | 5 NM, 3 HD, 3 HR, 4 DT, 4 FM, 1 TB |

Reviewer 2: ```suggestion
| Quarterfinals | 5 NM, 3 HD, 3 HR, 4 DT, 4 FM, 1 TB |
```

> | Semifinals    | 5 NM, 3 HD, 3 HR, 4 DT, 4 FM, 1 TB |

Reviewer 2: ```suggestion
| Semifinals | 5 NM, 3 HD, 3 HR, 4 DT, 4 FM, 1 TB |
```

> | Finals        | 5 NM, 3 HD, 3 HR, 4 DT, 4 FM, 1 TB |

Reviewer 2: ```suggestion
| Finals | 5 NM, 3 HD, 3 HR, 4 DT, 4 FM, 1 TB |
```

> | Grand Finals  | 5 NM, 3 HD, 3 HR, 4 DT, 4 FM, 1 TB |

Reviewer 2: ```suggestion
| Grand Finals | 5 NM, 3 HD, 3 HR, 4 DT, 4 FM, 1 TB |
```
